extensions/search-mdn/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "search-mdn",
+  "title": "Search MDN",
+  "description": "Search MDN Web Docs",
+  "icon": "icon.png",
+  "author": "krzysztofzuraw",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search MDN",
+      "subtitle": "",
```

Let's add the name of service here, feel free to choose MDN or Mozilla Developer Network here 😉 
```suggestion
      "subtitle": "MDN",
```